grid_utils/constant.h

```diff
@@ -16,5 +16,13 @@
  * You should have received a copy of the GNU Lesser General Public
  * License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
  **********************************************************************/
-#define RADIUS        (6371000.)
-#define STRING        255
+#define RADIUS   (6371000.)
+#define STRING   255
+
+#define EPSLN8   (1.e-8)
+#define EPSLN15  (1.0e-15)
+#define EPSLN30  (1.0e-30)
+#define EPSLN10  (1.0e-10)
+#define R2D (180/M_PI)
+#define TPI (2.0*M_PI)
+#define HPI (0.5*M_PI)
```

Comment:
Do you think one of the files in the constants directory would be a better place for these?

Reply:
I did think about it, but if people use create_xgrid via horiz_interp, it could potentially lead to answer changes... I'll make changes to use constants and see what other reviewers say

Reply:
Or maybe that could be a separate PR, it seems like we need to add another file/mod for constants for c.  it seems like we're not allowed to add bind(C) to parameters.